### ISSUE TITLE
fetch data once without listen

### ISSUE THREAD

**Issue by RahavLussato** [tiberiuc/redux-react-firebase#3]
hi i saw in the docs that this one is In the future (Firebase queries),
do you have any estimation for this one ?
after i tried couple of packages of react and firebase integration i can say that your's package is
the most efficient one because its include redux in perfect way, but fetching data is basic feature that is missing.

**Comment by tiberiuc** [comment]
thank you for the interest of this package, no I don't have a time estimate as  I still figure it out how to implement it.
about queries one of the ideas I like is something like that

```
@firebase( [
  'todos#orderByChild=time&startAt=2&limitToLast=5'
])
```

I would like to use `?` instead `#` but `?` it is a valid key character.
do you think something like that will work for you or if you have any other idea I'm open to take them in consideration

about fetching data only once instead listen, do you have any suggestion , maybe like a query with a param like `listen=false`?

**Comment by RahavLussato** [comment]
hi thanks for the fast replay,
for queries its looks like yours suggestion is the best one,
for fetching i think it should be done from function and not from decorator because you want the ability to call it from different places for example in events.

**Comment by tiberiuc** [comment]
you already have access to that, first you need to use the firebase decorator, that will add firebase as a prop to your component

```
@firebase()
```

then you need to get firebase from the props

```
const {firebase} = this.props
```

and then you will have `firebase.ref` which is the root reference in your firebase and you can do anything you want with it like 

```
firebase.ref.child('contacts').once('value', snap => console.log(snap.val())
```

is that what you need ?

**Comment by RahavLussato** [comment]
this one solve the fetch i needed Thanks :+1: 
now only missing the queries part

**Comment by tiberiuc** [comment]
I also responded about that in a different issue #4  and I will do it soon as I also need it in one of my projects

**Comment by RahavLussato** [comment]
:+1:  thanks

**Comment by RahavLussato** [comment]
i started to so some changes to support minimal queries, you can see my changes here: PR #6

**Comment by tiberiuc** [comment]
thank you very much, I merged your PR #6  and released a new version
